pages/linux/mono.md

```diff
@@ -0,0 +1,20 @@
+# mono
+
+> Mono, the open source development platform based on the .NET Framework.
```

Comment:
Perhaps we could make  this a bit clearer as to what it actually does?
```suggestion
> Runtime for the .NET Framework.
```
Since `mono` is the runtime itself, and `mcs` (now `csc`) is the compiler?